fog/view/connection/src/bin/router_client.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! A simple binary that creates a Fog View Router Client and sends off
+//! requests to the Fog View Router server.
+
+use mc_fog_uri::FogViewRouterUri;
+use mc_fog_view_connection::fog_view_router_client::FogViewRouterGrpcClient;
+use std::{str::FromStr, sync::Arc};
+
+fn main() {
+    futures::executor::block_on(async_main()).unwrap();
+}
+
+async fn async_main() -> Result<(), grpcio::Error> {
```

Comment:
`block_on` will execute each future on the current thread, i.e. talking to each shard in series. You'll want a [multi-threaded executor](https://docs.rs/tokio/latest/tokio/attr.main.html#multi-threaded-runtime) before long, so consider adding it now.

```suggestion
#[tokio::main(flavor = "multi_thread")]
async fn main() -> Result<(), grpcio::Error> {
```

Reply:
Done